general/src/pii/builtin.rs

```diff
@@ -566,32 +567,32 @@ mod tests {
     fn test_creditcard() {
         assert_text_rule!(
             rule = "@creditcard";
-            input = "John Appleseed 1234-1234-1234-1234!";
-            output = "John Appleseed ****-****-****-1234!";
+            input = "John Appleseed 4571234567890111!";
```

Comment:
👍 , but can we also check in a test for CC numbers with dashes (e.g. VISA/Amex) to check if the pattern replacement works.